pkg/service/credential/storage.go

```diff
@@ -223,16 +227,55 @@ func (cs *Storage) StoreCredential(ctx context.Context, request StoreCredentialR
 	return cs.storeCredential(ctx, request, credentialNamespace)
 }
 
-func (cs *Storage) StoreStatusListCredentialTx(ctx context.Context, request StoreCredentialRequest, tx storage.Tx) error {
-	wc, err := cs.getStoreCredentialWriteContext(request, statusListCredentialNamespace)
+func (cs *Storage) StoreStatusListCredentialTx(ctx context.Context, request StoreCredentialRequest, statusPurpose statussdk.StatusPurpose, tx storage.Tx) error {
```

Comment:
nit: may make sense for tx storage.Tx to come as the 2nd parameter here and elsewhere